### PR TITLE
Exercise question styling

### PR DESCRIPTION
**Pull request by jivewise**
screen shot 2015-02-13 at 4 40 00 am
screen shot 2015-02-13 at 4 40 13 am

Adding in styles for questions and answers.  Will need to check more question types later as we get access to them.

### CODE REVIEW

File: style/question.less [hunk 1]
```diff
@@ -86,40 +104,115 @@
...
+      margin: 0.5em 0;
+      padding-left: 0;
+      line-height: 1.5em;
+      display: inline-block;
       width: 100%;
+      float:left;
+      transition: background-color .2s ease-in;
+      transition: padding-left .2s ease-in;
+
+      //Materialize hack
```
Comment (philschatz) [inline style/question.less]: Are these hacks still used?

File: style/question.less [hunk 2]
```diff
@@ -86,40 +104,115 @@
...
+    &.has-correct-answer {
+      .answer-input-box { display: none; }
+
+      .answers-table .answers-answer .answer-label {
+        padding-left: 10px;
+      }
 
+      .answers-table .answer-correct:before,
+      .answers-table .answer-checked:not(.answer-correct):before { 
+        content:"\f00c";
```
Comment (philschatz) [inline style/question.less]: Can you pull this from the `@fa-*` variables from the `font-awesome.less`?

File: style/question.less [hunk 3]
```diff
@@ -1,63 +1,81 @@
...
+    line-height: 150%;
+  }
+
+  .free-response,
+  .free-response:before,
+  .question-feedback,
+  .question-feedback:before {
+    font-size: 0.75em;
+  }
+  .free-response:before {
```
Comment (philschatz) [inline style/question.less]: Could you put this inside the `.free-response {` selector?

```
.free-response {
  &::before {
    ...
  }
}
```

File: style/question.less (hunk 2 above)
Comment (jivewise) [inline style/question.less]: Ah, that's a good idea, I'll check.

File: style/question.less (hunk 1 above)
Comment (jivewise) [inline style/question.less]: I'm using the transitions, the Materialize hacks, I haven't tried taking them out, but I will to see if it makes a difference.

**Comment by jivewise** [comment]
screen shot 2015-02-13 at 5 30 48 am
Here's a screenshot of what it looks like with long answers and feedback.

File: style/question.less (hunk 3 above)
Comment (jivewise) [inline style/question.less]: Yep, just did that.  Thanks for the suggestion.

**Comment by jivewise** [comment]
Uploading a screenshot of the answering state

screen shot 2015-02-13 at 6 01 43 am

**Pull request merged**